### DISCUSSION TITLE
Biblioteca time

### DISCUSSION THREAD

**Discussion by ignaciomanzanares** [IIC2133-PUC/2024-1#96] [Tarea 1] [answered]
Se puede usar la biblioteca time.h?

**Comment by benjavicente** [comment]
¿Para qué? 🤔

**Reply by ignaciomanzanares** [reply]
Para fijar los tiempos de llegada de las ordenes en la parte 1, en caso de empate.

**Reply by benjavicente** [reply, marked as answer]
Sí, pero no es necesario el tiempo literal para la tarea 😅
